doc/project/checklist.md

```diff
@@ -247,46 +247,55 @@ Ensure all these tests verify back-2back access with zero delays, along with par
 
 ### SIM_ALT_TOOL_SETUP
 
-Verify that the sanity test passes cleanly (with no warnings) with one additional tool apart from the primary tool selected for signoff.
+Verify that the smoke test passes cleanly (with no warnings) with one additional tool apart from the primary tool selected for signoff.
 
-### SIM_SANITY_REGRESSION_SETUP
+### SIM_SMOKE_REGRESSION_SETUP
 
-Sanity regression set up for code health. A small suite of tests identified for running the sanity regression on. If the testbench has more than one compile-time configuration, then a sanity test for each configuration should be ideally selected.
+Smoke regression set up for code health.
+A small suite of tests must be identified for the smoke regression.
+If the testbench has more than one build configurations, then each configuration must have a representative test added to the smoke regression.
 
 ### SIM_NIGHTLY_REGRESSION_SETUP
 
-Nightly regression for running all tests with multiple random seeds (iterations) setup. Selecting the number of iterations for running each test is subjective - it depends on the failure rate and available compute resources. For starters, it is recommended to set iterations to 100 for each test. It may be trimmed down if the compute load is too high. As such, the goal should be to have the nightly regression finish overnight so that the results are available next morning for triage.
+Nightly regression for running all tests with multiple random seeds (iterations) setup.
+Selecting the number of iterations for running each test is subjective - it depends on the failure rate and available compute resources.
+For starters, it is recommended to set iterations to 100 for each test.
+It may be trimmed down if the compute load is too high.
+As such, the goal should be to have the nightly regression finish overnight so that the results are available next morning for triage.
```

Comment:
In the rest of this document, bullet points are used for a list of things to do. Here, you're using them as sentence breaks. I think this should probably be a single bullet point with multiple sentences.

Reply:
Gah! I was reading the "+" signs in the GitHub diff as bullet point separators. Sorry for the noise.